src/test/resources/ssl/reload/node-new.crt.pem

```diff
@@ -0,0 +1,88 @@
+Bag Attributes
```

Comment:
why do we need these attributes separately? Can't we read them from cert using ssl library?

Reply:
Bag attributes in OpenSSL PEM files seem to be a side effect of converting a PKCS#12 keypair into PEM format.